src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -455,15 +458,32 @@ bool AdaptDispatch::CursorSavePosition()
 // - True if handled successfully. False otherwise.
 bool AdaptDispatch::CursorRestorePosition()
```

Comment:
Now this almost seems like it needs to be renamed `CursorRestoreState`

Reply:
Agreed. I'll do that.